gateway/api/ray.py

```diff
@@ -77,7 +77,10 @@ def create_ray_cluster(
     cluster_name = cluster_name or f"{user.username}-{str(uuid.uuid4())[:8]}"
     if not cluster_data:
         cluster = get_template("rayclustertemplate.yaml")
-        cluster_data = yaml.safe_load(cluster.render({"cluster_name": cluster_name}))
+        manifest = cluster.render(
+            {"cluster_name": cluster_name, "user_id": user.username}
```

Comment:
thinking out loud... is there any chance that `user` could be `None` here?

Reply:
only if job has no user associated to it, which is not possible as all jobs are authored by authentication, which is only way to access endpoints

Reply:
nah, this was a dead end